lib/colorado_lottery.rb

```diff
@@ -0,0 +1,34 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants, :winners, :current_contestants
+  def initialize()
+    @registered_contestants = Hash.new { |hash, key| hash[key] = [] }
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(person_interested, game_interest)
+    person_interested.age >= 18 &&
+    person_interested.game_interests.include?(game_interest.name)
+  end
+
+  def can_register?(person, game)
+    if interested_and_18?(person, game) && game.national_drawing?
```

Comment:
No need for control flow here. We only ever need a condition for a boolean method. 
```
interested_and_18?(contestant, game) &&
(game.national_drawing? || !contestant.out_of_state?)
```